src/renderer/Components/CreatingPackage/CreatingOrder/FilePicker/index.jsx

```diff
@@ -10,8 +10,15 @@ function FilePicker() {
 
   const setFileName = (event) => {
```

Comment:
```suggestion
  const setFileInfo = (event) => {
```
해당 함수에서 더 이상 fileName 만을 설정하고 있진 않은 거 같습니다.

Reply:
수정하겠습니다😁